packages/contracts-bedrock/test/opcm/DeployImplementations.t.sol

```diff
@@ -383,68 +401,59 @@ contract DeployImplementations_Test is Test {
     function test_v2ParamsValidation_withHugeValues_reverts() public {
         // When V2 flag is enabled, huge V2 params should be rejected
         DeployImplementations.Input memory input = defaultInput();
-        input.devFeatureBitmap = DevFeatures.DEPLOY_V2_DISPUTE_GAMES; // V2 enabled
+        bool shouldRevert = isDevFeatureEnabled(DevFeatures.DEPLOY_V2_DISPUTE_GAMES);
```

Comment:
Would it be simpler to add a skip if `DEPLOY_V2_DISPUTE_GAMES`. Rather than branching for each check. This function doesn't appear to be appropriate for a v1 deployment.

Reply:
Went ahead and simplified [here](https://github.com/ethereum-optimism/optimism/pull/17985/commits/34d10ff7cc4c9f3d5d6409f8d2cd9e89142cd62b)

Reply:
oh I see. I didn't see realize you intended to test the v1 case with this [line](https://github.com/ethereum-optimism/optimism/pull/17985/files#diff-7a2c45858003c98dac33a03b369e30a57d0d622afb0d54351d7c511e7bbc7d8dL386) removed. Either branching or having a separate function is OK. Though if we want to also check that no reverts occur in v1, we should probably give the test a better name.

Reply:
Added a separate test - it'll make the clean up simpler to split them up.